src/explore-education-statistics-admin/src/queries/methodologyQueries.ts

```diff
@@ -0,0 +1,13 @@
+import { createQueryKeys } from '@lukemorales/query-key-factory';
+import methodologyService from '@admin/services/methodologyService';
+
+const methodologyQueries = createQueryKeys('methodology', {
+  getMethodologyStatuses(methodologyId: string) {
+    return {
+      queryKey: ['getMethodologyStatuses', methodologyId],
```

Comment:
You don't need the `getMethodologyStatuses` part. If this has been copied from `releasePermissionQueries`, would suggest updating them to remove their `listReleaseBlah` strings as well